README.md

```diff
@@ -47,6 +43,23 @@ pam-ad sends information about ad changes to the Kafka topic `stilling-intern` t
 [navikt/pam-puls](https://github.com/navikt/pam-puls) stores information about ad statistics and sends the
 information to the Kafka topic `puls` that pam-import-api listens to.
 
+# Secrets
+
+All secrets are now available in the Nais console. There are three of them:
+* pam-import-api-script-secret is not used by the application, but should be used by developers when they need to use the provider-scripts
+* import-api-dev-key contains information about a provider in dev that developers can use to test the API.
+* pam-import-api-env-env-secret contains the JWT secret needed to verify tokens sent in. This is the only secret used by the application.
+
+## pam-import-api-env-env-secret
```

Comment:
Har denne `env` som en del av navnet to ganger, eller er det den som heter `pam-import-api-env-secret`?

Reply:
Ooops, good catch!